esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -1751,6 +1751,19 @@ bool addWUQueryFilterTime(WUSortField *filters, unsigned short &count, MemoryBuf
     return true;
 }
 
+bool addWUQueryFilterTotalClusterTime(WUSortField& sortorder, WUSortField *filters, unsigned short &count, MemoryBuffer &buff, unsigned milliseconds, WUSortField value)
+{
+    if (milliseconds == 0)
+        return false;
+
+    VStringBuffer vBuf("%d", milliseconds);
+    filters[count++] = value;
+    buff.append(vBuf);
```

Comment:
Might be better to have buff as "filterbuf" like the caller, just to make it obvious.